docs/content/docs/wip/ingress.md

```diff
@@ -0,0 +1,390 @@
+
+1. Deploy the `httpbin` service into the `httpbin` namespace after enrolling its namespace to the mesh. The `httpbin` service runs on port `14001`. 
+    ```bash
+    # Create the httpbin namespace
+    kubectl create namespace httpbin
+
+    # Add the namespace to the mesh
+    osm namespace add httpbin
+
+    # Deploy httpbin service in the httpbin namespace
+    kubectl apply -f docs/example/manifests/samples/httpbin/httpbin.yaml -n httpbin
+    ```
+
+    Confirm the `httpbin` service and pods are up and running.
+
+    ```console
+    $ kubectl get pods -n httpbin
+    NAME                       READY   STATUS    RESTARTS   AGE
+    httpbin-74677b7df7-zzlm2   2/2     Running   0          11h
+    ```
+
+    ```console
+    $ kubectl get svc -n httpbin
+    NAME      TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)     AGE
+    httpbin   ClusterIP   10.0.22.196   <none>        14001/TCP   11h
+    ```
+
+1. Apply an ingress configuration yaml to expose the HTTP path `/status/200` on the `httpbin` service. 
+    ```bash
+    kubectl apply -f docs/example/manifests/samples/ingress-httpbin.yaml -n httpbin
+    ```
+
+    Confirm that the httpbin-ingress has been successfully deployed. 
+    
+    ```console
+    $ kubectl get ingress -n httpbin
+    NAME              CLASS    HOSTS   ADDRESS         PORTS   AGE
+    httpbin-ingress   <none>   *       20.72.132.186   80      11h
+    ```
+
+1. Confirm that a request to the httpbin service from the external IP address of the Ingress resource succeeds (in this case, the external address would be `20.72.132.186`)
+
+    ```bash
+    curl http://<external-ip>/status/200
+    ```
+
+
+## Other ingress configuration examples
+
+The following section describes sample ingress configurations used to expose services managed by OSM outside the cluster. The configuration might differ based on the ingress controller being used.
+
+The example configurations describe how to expose HTTP and HTTPS routes for the `bookstore-v1` service running on a pod with the service acccount `bookstore-v1` on port `14001` in the `bookstore` namespace, outside the cluster. The ingress configuration will expose the HTTP path `/books-bought` on the `bookstore-v1` service.
+
+Since OSM uses its own root certificate, the ingress controller must be provisioned with OSM's root certificate to be able to authenticate the certificate presented by backend servers when using HTTPS ingress. With `Tresor` as the certificate provider, OSM stores the CA root certificate in a Kubernetes secret named `osm-ca-bundle` with the key `ca.crt` in the namespace OSM is deployed (`osm-system` by default). When using other certificate providers such as `cert-manager.io` or `Hashicorp Vault`, the `osm-ca-bundle` secret must be created by the user with the base64 encoded root certificate stored as the value to the `ca.crt` attribute in the secret's data.
```

Comment:
thanks for this, this was missing since one of my changes for HA

Reply:
I'm going to change the examples to use the httpbin service (like I did in the demo) instead of bookstore